### PR TITLE
Update CHANGE_LOG for 0.53.0

### PR DESCRIPTION
**Pull request by stuartarchibald**
As title.

### CODE REVIEW

**Comment by stuartarchibald** [comment]
CC @numba/czars

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,3 +1,227 @@
...
+  compare-and-swap is extended to support 64-bit integers (Michael Collison).
+* Addition of ``cuda.is_supported_version()`` to check if the CUDA runtime
+  version is supported (Graham Markall).
+* The CUDA dispatcher now shares infrastructure with the CPU dispatcher,
+  improving launch times for lazily-compiled kernels (Graham Markall).
+* The CUDA Array Interface is updated to version 3, with support for streams
+  added (Graham Markall).
+* Tuples and ``namedtuples`` can now be passed to kernels (Graham Markall).
+* Initial support for Cooperative Groups is added, with support for Grid Groups
+  and Grid Sync (Graham Markall).
```
Comment (gmarkall) [inline CHANGE_LOG:38]: Just realised from reading the comprehensive list below that this should be:

```suggestion
  and Grid Sync (Graham Markall and Nick White).
```

**Review by gmarkall — CHANGES_REQUESTED** [review]
Looks good to me in general - one suggestion to add credit to Nick White for CGs in the CUDA highlights as well (which I realised when reading the full changelog).

File: CHANGE_LOG (hunk 1 above)
Comment (stuartarchibald) [reply CHANGE_LOG:38]: ah yes, well caught, didn't update both places.

Comment (stuartarchibald) [reply CHANGE_LOG]: change accepted, thanks.

**Comment by stuartarchibald** [comment]
> Looks good to me in general - one suggestion to add credit to Nick White for CGs in the CUDA highlights as well (which I realised when reading the full changelog).

Thanks for the review @gmarkall

**Comment by stuartarchibald** [comment]
5a65dfe  is an updated to include https://github.com/numba/numba/pull/6685 which is needed for 0.53 as it adds the Python 3.9 classifiers to setup.py.

**Review by sklam — APPROVED** [review]
LGTM

**Pull request merged**